packages/core/src/routes/connector.test.ts

```diff
@@ -89,6 +92,25 @@ describe('connector route', () => {
       expect(response).toHaveProperty('statusCode', 400);
     });
 
+    it('throws if more than one same-platform social connectors sharing the same `target`', async () => {
+      getLogtoConnectorsPlaceHolder.mockResolvedValueOnce([
+        {
+          dbEntry: mockConnector0,
+          metadata: { ...mockMetadata0, target: 'target' },
+          type: ConnectorType.Social,
+          ...mockLogtoConnector,
+        },
+        {
+          dbEntry: mockConnector1,
+          metadata: { ...mockMetadata1, target: 'target' },
+          type: ConnectorType.Social,
+          ...mockLogtoConnector,
+        },
+      ]);
+      const response = await connectorRequest.get('/connectors').send({});
+      expect(response).toHaveProperty('statusCode', 400);
```

Comment:
should the response code be 400?

Reply:
This aligns with other checks (no more than one sms/email connector is accepted).